### PR TITLE
Aligned named with hypenated widget names

### PR DESCRIPTION
**Pull request by tomdye**
**Type:** feature

The following has been addressed in the PR:

* [ ] There is a related issue
* [ ] All code has been formatted with [`prettier`](https://prettier.io/) as per the [readme code style guidelines](./../#code-style)
* [ ] Unit or Functional tests are included in the PR

**Description:**

Aligns with https://github.com/dojo/widgets/pull/483

### CODE REVIEW

File: dojo/index.ts [hunk 1]
```diff
@@ -1,49 +1,49 @@
-import * as accordionPane from './accordionPane.m.css';
+import * as accordionPane from './accordion-pane.m.css';
 import * as button from './button.m.css';
 import * as calendar from './calendar.m.css';
 import * as checkbox from './checkbox.m.css';
 import * as comboBox from './comboBox.m.css';
-import * as enhancedtextinput from './enhancedtextinput.m.css';
+import * as enhancedtextinput from './enhanced-text-input.m.css';
```
Comment (dylans) [inline dojo/index.ts]: should this be enhancedTextInput for consistency?

File: dojo/index.ts [hunk 2]
```diff
@@ -1,49 +1,49 @@
...
 import * as slider from './slider.m.css';
-import * as splitPane from './splitPane.m.css';
-import * as tabController from './tabController.m.css';
-import * as textarea from './textarea.m.css';
-import * as textinput from './textinput.m.css';
-import * as timePicker from './timePicker.m.css';
-import * as titlePane from './titlePane.m.css';
+import * as splitPane from './split-pane.m.css';
+import * as tabController from './tab-controller.m.css';
+import * as textarea from './text-area.m.css';
```
Comment (dylans) [inline dojo/index.ts]: should this be textArea for consistency?

File: dojo/index.ts [hunk 3]
```diff
@@ -1,49 +1,49 @@
...
+import * as textinput from './text-input.m.css';
```
Comment (dylans) [inline dojo/index.ts]: should this be textInput for consistency?

**Review by dylans — COMMENTED** [review]
Change looks fine, a couple of comments about consistency of the camelCasing of import names.

File: dojo/index.ts [hunk 4]
```diff
@@ -1,49 +1,49 @@
-import * as accordionPane from './accordionPane.m.css';
+import * as accordionPane from './accordion-pane.m.css';
 import * as button from './button.m.css';
 import * as calendar from './calendar.m.css';
 import * as checkbox from './checkbox.m.css';
 import * as comboBox from './comboBox.m.css';
```
Comment (dylans) [inline dojo/index.ts]: Any reason we kept this as comboBox?

**Review by dylans — CHANGES_REQUESTED** [review]

File: dojo/index.ts (hunk 4 above)
Comment (tomdye) [reply dojo/index.ts]: perhaps because i missed it, will look into it

**Review by dylans — APPROVED** [review]

**Pull request merged**